### PR TITLE
fix: sanitize HTML in release body

### PR DESCRIPTION
**Pull request by pierreneter**
Related to issues: https://github.com/electron/releases/issues/17 https://github.com/electron/electronjs.org/issues/1296 
and PR: https://github.com/electron/releases/pull/19

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -13,7 +13,8 @@ const renderer = remark()
   .use(autolinkHeadings, {behaviour: 'wrap'})
   .use(inlineLinks)
   .use(emoji)
-  .use([hljs, html], {sanitize: false})
+  .use(hljs)
+  .use(html, {sanitize: true})
```
Comment (zeke) [inline index.js]: Test are failing. Maybe the `[hljs, html]` array needs to be preserved?

Comment (pierreneter) [reply index.js]: so you want: `[hljs, html], {sanitize: false}` right?

Comment (pierreneter) [reply index.js]: so you want: `[hljs, html], {sanitize: true}` right?

**Comment by zeke** [comment]
https://github.com/electron/electronjs.org/issues/1296 was resolved by editing the contents of the changelog entry, so this change is not need any more for that specific issue. It would be handy to allow optional sanitization to hubdown, but it's no longer specifically needed by the Electron team, as we don't use this module to parse any untrusted content.

@pierreneter if you want to continue adding this feature for general purpose use, I would suggest making it an option at the top-level of hubdown itself, like:

```js
hubdown(markdownString, {sanitize: true})
```

Then passing that configuration option to the `html` invocation. That will give users flexibility to choose how they want to sanitize, and it wouldn't be a breaking change.

Also, your initial code that looked like this was correct:

```
  .use(hljs)
  .use(html, {sanitize: true})
``` 

Sorry for misleading you there! The reason it failed tests was because using the [default GitHub sanitization options](https://github.com/syntax-tree/hast-util-sanitize#schema) causes the DOM ids to have a `user-content-` prefix added. Example:

```html
<h2 id="user-content-sanitized-fixture"><a href="#sanitized-fixture">Sanitized Fixture</a></h2>
```

**Comment by zeke** [comment]
Based on my last comment and the age of this PR, I'm gonna go ahead and close it out. Feel free to comment here if I've closed it in error.

**Pull request closed without merging**